setup.py

```diff
@@ -55,7 +55,7 @@
 ]
 
 # cython extensions
-ext_compiler_args = ["-std=c++11"]
+ext_compiler_args = ["-std=c++11", "-O2"]
```

Comment:
cython automatically uses an optimization flag when compiling. So, will not have speedup for this. And, couldn't find any big difference between -O2 and -O3 at this time.